editor/gui/scene_tree_editor.h

```diff
@@ -201,7 +201,7 @@ class SceneTreeDialog : public ConfirmationDialog {
 	static void _bind_methods();
 
 public:
-	void popup_scenetree_dialog();
+	void popup_scenetree_dialog(Node *p_selected_node = nullptr, Node *p_marked_node = nullptr, bool p_marked_node_selectable = false, bool p_marked_node_children_selectable = true);
```

Comment:
I find the default value of `p_marked_node_selectable` odd. It's `false`, but you pass `true` in almost every case, or not pass any marked node. Thus the default is effectively used only once, explicitly.

Reply:
Agreed. I took the same defaults as the `set_marked` method's defaults.
Did not want to deviate from that, but also did not want to change set_marked's defaults since that's a breaking change I imagine.

Reply:
SceneTreeEditor is not exposed, so you can freely change it.